### ISSUE TITLE
Why repeat data_shot

### ISSUE THREAD

**Issue by zfbz** [icoz69/DeepEMD#51]
Hi, I'm a beginner. I want to know why to repeat data_shot during meta-training. 
train_meta.py, line.158
`logits = model((data_shot.unsqueeze(0).repeat(num_gpu, 1, 1, 1, 1), data_query))`

**Comment by icoz69** [comment]
hi, this is for multi-gpu computing.

**Comment by zfbz** [comment]
Why does `data_query` not need to repeat(only `data_shot`)? Are there any related multi-gpu programming references? I haven't used this before. Thanks.
